workflow-to-js-transformer/src/main/resources/Workflow.xslt

```diff
@@ -295,11 +294,8 @@
                 updateActionStatus(document);
                 return ALREADY_EXECUTED;
             }
-            // propagate the post-processing field to response custom data if it exists on the document custom data
+            // propagate the custom data if it exists
             var responseCustomData = actionDetails.customData ? actionDetails.customData : {};
-            if(!isEmpty(document.getCustomData(postProcessingScriptPropertyName))){
```

Comment:
Some of the unit tests were checking for this and are now failing. Update them to look at the Scripts set instead.

Reply:
@mmaBel Updated to address these tests